src/annotator/anchoring/test/types-test.js

```diff
@@ -19,43 +21,290 @@ describe('types', function () {
     document.body.appendChild(container);
   });
 
-  after(function () {
+  afterEach(() => {
     container.remove();
   });
 
-  describe('TextQuoteAnchor', function () {
-    describe('#toRange', function () {
-      it('returns a valid DOM Range', function () {
+  describe('RangeAnchor', () => {
+    let fakeSniff;
+    let fakeSerializedRange;
+    let fakeNormalize;
+    let fakeSerialize;
+
+    beforeEach(() => {
+      fakeSerializedRange = sinon.stub();
+      fakeSerialize = sinon.stub();
+      fakeNormalize = sinon.stub().returns({
+        serialize: fakeSerialize,
+      });
+      fakeSniff = sinon.stub().returns({
+        normalize: fakeNormalize,
+      });
+      $imports.$mock({
+        './range': {
+          sniff: fakeSniff,
+          SerializedRange: fakeSerializedRange,
+        },
+      });
+    });
+
+    afterEach(() => {
+      $imports.$restore();
+    });
+    function createRangeAnchor() {
```

Comment:
This function name is confusing. It creates a DOM `Range` rather than a `RangeAnchor` as the name suggests. Given how simple it is, it looks like it would be just as easy to replace the callers with `new Range()`.